e2e-test/src/test/resources/pluginParameters.properties

```diff
@@ -48,3 +48,23 @@ filePluginOutputFolder=file-plugin-output
 skipHeaderTrue=true
 fileSinkPathSuffix=yyyy-MM-dd-HH-mm-ss
 ## FILE-PLUGIN-PROPERTIES-END
+
+## DateTransform-PLUGIN-PROPERTIES-START
+dateTransformBQCreateTableQueryFile=testdata/BigQuery/bqDateTransformCreateTableQuery.txt
+dateTransformBQInsertDataQueryFile=testdata/BigQuery/bqDateTransformInsertQuery.txt
+dateTransformBQCreateTableQueryFile1=testdata/BigQuery/bqDateTransformCreateTableLong.txt
+dateTransformBQInsertDataQueryFile1=testdata/BigQuery/bqDateTransformInsertQueryLong.txt
+dateTransformGcsCsvFile=testdata/DateTransformGCS/gcsDateTransform.csv
+dateTransformGcsCsvFileSchema=[{"key":"id","value":"int"},{"key":"name","value":"string"},{"key":"date_of_birth","value":"int"}
+sourceFieldName=date_of_admission
+targetFieldName=date_of_admission
+sourceFieldDateFormat=yyyy-mm-dd
+targetFieldDateFormat=dd-mm-yyyy
+SourceFieldName=transaction_date,date_of_admission,date_of_joining
+TargetFieldName=transaction_date,date_of_admission,date_of_joining
+SourceFieldDateFormat=yyyy-mm-dd
+TargetFieldDateFormat=dd-mm-yyyy
+IncorrectSourceFieldName=@#$%^&
+IncorrectTargetFieldName=@#$%^&
+targetfieldNameError=transaction_date
```

Comment:
no need of this -> instead u can use "Single field in source" and "multiple fields property in target" to get error.

Reply:
@sawantpritam changes done